src/Recyclarr.Tests/Config/Services/ConfigurationLoaderTest.cs

```diff
@@ -151,4 +152,31 @@ public void Validation_success_does_not_throw(ConfigurationLoader<TestConfig> co
         Action act = () => configLoader.LoadFromStream(new StringReader(testYml), "fubar");
         act.Should().NotThrow();
     }
+
+    [Test, AutoMockData(typeof(ConfigurationLoaderTest), nameof(BuildContainer))]
+    public void Test_secret_loading(ConfigurationLoader<SonarrConfiguration> configLoader,
+        [Frozen(Matching.ImplementedInterfaces)] MockFileSystem fs,
```

Comment:
@rcdailey  -- this is where I'm struggling.  I'm not sure how to properly initialize the MockFileSystem object.  Probably could figure it out but was hoping you could just point me in the right direction.